### ISSUE TITLE
FYI: NEWS.md is not found by news()

### ISSUE THREAD

**Issue by HenrikBengtsson** [dynverse/princurve#13]
I noticed you Markdown-ified NEWS to NEWS.md.  Several packages do this and CRAN parses it and renders it in HTML online.  Unfortunately, it also breaks `news()` who will not find it, cf. https://github.com/HenrikBengtsson/Wishlist-for-R/issues/40;
```r
> news(package = "Seurat")
NULL
```

**Comment by HenrikBengtsson** [comment]
Maybe it's possible to have a copy `NEWS` of `NEWS.md` and keep both under `inst/`.

**Comment by rcannood** [comment]
Yes; I noticed this too, but preferred to have the news more legible on GitHub than have it work with `news()`. Although you're right, there's no reason why I shouldn't make both work.

What do you think about this solution? [feature/improve_news](https://github.com/dynverse/princurve/tree/feature/improve_news) ([Diff](https://github.com/dynverse/princurve/compare/feature/improve_news)). Would this be the most useful solution for both users and developers?

By the way, I really like Wishlist-for-R :+1: I wish more issues were closed, though.
